target/linux/ar71xx/base-files/etc/board.d/02_network

```diff
@@ -191,7 +191,11 @@ ar71xx_setup_interfaces()
 	all0315n|\
 	dlan-hotspot|\
 	dlan-pro-500-wp|\
-	ens202ext|\
+	ens202ext)
```

Comment:
@mpratt14 This approach for fixing this issue for ens202ext is wrong, as you're effectively breaking switch configuration for other devices.

Reply:
Can you please explain? By "other devices" do you mean other boards, and how would my commit affect any other board in the list?

If you're talking about existing ENS202EXT that other people have, I still can't see the consequence of this change. Even if this would be applied because of a sysupgrade image and nvram/config is to be kept, it would be a step someone else would otherwise have to do manually. Ethernet was nonexistent until I did exactly this and then regenerate board.json. Is that not the reason that there are other customized entries for other boards? Is that not the reason this file exists in the build?

If I'm mistaken please fully inform me,
Thanks

Reply:
@mpratt14 Situation before your patch was, that all0258n, all0315n, dlan-hotspot, dlan-pro-500-wp, ens202ext, ja76pf2, rocket-m-ti, ubnt-unifi-outdoor devices would all use `ucidef_set_interface_lan "eth0 eth1"`

Situation after your patch is, that all0258n, all0315n, dlan-hotspot, dlan-pro-500-wp and ens202ext would use your new switch/vlan config, and ja76pf2, rocket-m-ti, ubnt-unifi-outdoor would use the `ucidef_set_interface_lan "eth0 eth1"`.

In other words, your patch as it is, is probably going to break network setup on all0258n, all0315n, dlan-hotspot and dlan-pro-500-wp devices.

Is that more clear now?

Reply:
Sorry, I'm pretty new with all this. Thanks for expanding but that isn't really an explanation of why this is true. After looking very carefully I recognized the or operator "|" and I see now that this is a list of very large if-then statements, and the target ENS202EXT was already part of a statement with other routers in it, and that my commit made the statement end early with different information....

About to pass another commit to put the target at the end before the wildcard.

Thanks